src/VisualStudio/Core/Test/ProjectSystemShim/VisualBasicHelpers/MockCompilerHost.vb

```diff
@@ -20,6 +20,12 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim.Vi
             End Get
         End Property
 
+        Public Shared ReadOnly Property NoSdkCompilerHost As MockCompilerHost
```

Comment:
Well, if they violated the spec we were already broken with them, and this won't make it any better. And with any luck, this code will cease to exist by the next release.